src/components/ui/voting-card/index.tsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
```

Comment:
Voting card components will be used only in one place and can be placed nearest HomeGovernance component